### DISCUSSION TITLE
Linked Repositories in Projects Beta?

### DISCUSSION THREAD

**Discussion by evandam** [community/community#5867] [Projects and Issues]
Hi folks,

Will linked repositories be supported in the new Projects beta? It seems difficult to add existing issues to a project besides manually adding each one individually. Being able to more easily (or automatically) add them from the project view or in bulk another way would make things much easier.

Thanks!

**Comment by StephenWithPH** [comment]
Also many requests for this behavior here: https://github.com/github/feedback/discussions/8884
